core/main.ts

```diff
@@ -147,29 +137,54 @@ function loadActionConfigs(session: Session, filePaths: string[]) {
           actionConfig.target.name = fileNameAsTargetName;
         }
 
+        // Users define file paths relative to action config path, but internally and in the
+        // compiled graph they are absolute paths.
+        actionConfig.fileName =
+          actionConfigsPath.substring(0, actionConfigsPath.length - "actions.yaml".length) +
```

Comment:
NIT: put `actions.yaml` in some constant.

Reply:
Fair shout, I'll do this in a later follow up PR with all names